README.md

```diff
@@ -139,7 +138,7 @@ This is very similar to how [Plug Router](https://github.com/elixir-plug/plug#th
 
 ## Runtime middleware
 
-All HTTP functions (`get`, `post`, etc.) can take a dynamic client as the first argument.
+All HTTP functions (`get`, `post`, and etc) can take a dynamic client as the first argument.
```

Comment:
Incorrect. `a, b, etc.` is the correct form (no `and`, and `etc.` not `etc` since `.` is for abbreviation)

https://www.grammarly.com/blog/et-cetera-etc/
https://www.chicagomanualofstyle.org/qanda/data/faq/topics/Punctuation/faq0040.html

Probably we may rewrite this like this:

> All HTTP functions, such as `Tesla.get/3` and `Tesla.post/4`, can take a ...

Reply:
@chulkilee You're right! Appreciate it.